scripting/lua/api/BonusSystem.h

```diff
@@ -54,7 +54,7 @@ class BonusListProxy : public SharedWrapper<const BonusList, BonusListProxy>
 
 	static const std::vector<typename Wrapper::CustomRegType> REGISTER_CUSTOM;
 
-	static std::shared_ptr<const Bonus> index(std::shared_ptr<const BonusList> self, int key);
+	static std::shared_ptr<const Bonus> index(std::shared_ptr<const BonusList> self, int key); //NOLINT
```

Comment:
Again, comments that are not comment but attributes...

Reply:
Removed temporarily.